element_optogenetics/optogenetics.py

```diff
@@ -134,46 +134,44 @@ class Sine(dj.Part):
 
 @schema
 class OptoStimParams(dj.Manual):
-    """A single opto stimulus that repeats
+    """A single optical stimulus that repeats.
 
     Power and intensity are both nullable. Users may wish to document one or the other.
 
     Attributes:
         opto_params_id (smallint): Stimulus parameter ID
         OptoWaveform (foreign key): OptoWaveform primary key
-        Device (foreign key): Device primary key
-        wavelength (smallint): Wavelength in nm of photo stim. light
+        wavelength (int): Wavelength in nm of optical stimulation light
         power ( decimal(6, 2), nullable ): Total power in mW from light source
-        intensity ( decimal(6, 2), nullable ): Power for given area
+        light_intensity ( decimal(6, 2), nullable ): Power for given area
```

Comment:
hm.. The documentation and declaration will start going out of sync. Let's keep thinking of a way to unify the table declaration into the documentation more organically with a single source of truth.

Reply:
I agree that this would be very useful. I think it might involve a significant amount of dev to fork the [docstring capture tool we use](https://github.com/oprypin/mkdocs-gen-files), and add a version of the table definition to it that would render as a table in the resulting site

Reply:
This would be great.  Let's work with the Engineering Team to add this functionality.